packages/dd-trace/src/ritm.js

```diff
@@ -60,7 +60,12 @@ module.exports = function hook (modules, onrequire) {
       if (modules && modules.indexOf(name) === -1) return exports // abort if module name isn't on whitelist
 
       // figure out if this is the main module file, or a file inside the module
-      const res = Module._findPath(name, [basedir, ...Module._resolveLookupPaths(name, this, true)])
+      const paths = Module._resolveLookupPaths(name, this, true)
```

Comment:
Looks like the original `require-in-the-middle` uses [resolve.sync](https://github.com/elastic/require-in-the-middle/blob/v2.2.2/index.js#L65) here. Would it still result in `null` or would it allow us to actually support the file? Looking at the code it seems to handle a lot more cases than `Module._resolveLookupPaths` alone.

Reply:
> Would it still result in null or would it allow us to actually support the file?

From what I can tell, it would actually _throw_ here, and the catch that and return the existing exports.

Reply:
Since it handles more cases, it's possible it wouldn't throw for this case specifically.

Reply:
Let's revisit this later, especially since PnP support might change completely the logic of this file.